### DISCUSSION TITLE
CP2K on the Hawk system at HLRS in Germany

### DISCUSSION THREAD

**Discussion by mattatlincoln** [cp2k/cp2k#1602] [General]
Has anyone run CP2K on the Hawk system at HLRS in Germany? People from our national supercomputer centre are trying to run some benchmarks there and hitting some issues.
Just trying this feature out...not sure if this if appropriate here.
Matt

**Comment by alazzaro** [comment]
Hello @mattatlincoln , this request got my attention... do you know what kind of errors they see? (I don't have access to the system, but I can investigate within HPE...)

**Reply by mattatlincoln** [reply]
Thanks Alfio,
" I have recently got access to run some benchmarks and am running into issues getting the code working on the system (I suspect due to problems with MPI+OpenMP in the HPE MPT MPI library). If you know of anyone who might have successfully compiled and got it working on the system, it would be useful to chat to them."
Could I put you or someone in touch with Andy at EPCC? I can get logs etc

**Reply by alazzaro** [reply]
Well, the right procedure would be to go via the HLRS support team... Definitely, you can give my name, but I don't have experience with that system, so I would prefer to stay under the shadow...

**Reply by mattatlincoln** [reply]
Ok, thanks.

**Reply by aproeme** [reply]
I believe @holly-t has been gaining experience with CP2K on HAWK, I'll close the circle through email :-)
